### DISCUSSION TITLE
How to tell SSS to 1 - increase recording time / 2 - use cache recording

### DISCUSSION THREAD

**Discussion by pinomat81** [djdd87/SynoAI#97] [General]
Hello,

First, I would like to thank you, this is really a reliable motion detection. I tested Reolink and SSS motion detection and it was crap... I am still waiting to test when it is raining at night.

I have two questions about what we can do with SynoAI and don't hesitate to tell me if I am doing something wrong :
1) Currently I use my camera motion detection instead of SSS (for CPU questions). When a motion is detected, SSS sends notifications to SynoAI which get/sends images to deepstack then tells SynoAI if something has been really detected (a person, a car, a girafe? :) ) Then I use a webhook+SSS Action rule external event to tell SSS to start recording for 20 seconds. But, it stops just about 20 seconds after (this is expected though). Then it detects another a new motion and start again to record 20 sec. In fact, this is the same person / car that triggered the motion detection system. 
Is that possible to only tell SSS to increase 20 sec the current recording with the action rule ? Maybe this works like that and I need to increase time recording to 30s ?

Second question  :
When we use integrated SSS for motion detection, it uses a cache to record x sec before every motion detection. Is there a way in Action Rule to tell to uses this cache ? 

Thank you

**Comment by djdd87** [comment]
Hey on both questions I don't know if that's possible. If the options aren't there then probably not - but you'd be best off asking SSS related questions on the Synology forums or Reddit. 

Sorry I can't help.

**Comment by pinomat81** [comment]
I will do that ;) And also check SSS API documentation. Thank you.
